packages/features/apps/AdminAppsList.tsx

```diff
@@ -183,26 +185,36 @@ const querySchema = z.object({
 const AdminAppsList = ({
   baseURL,
   className,
+  fromAdmin,
+  currentStep,
+  setIsLoading,
   useQueryParam = false,
 }: {
   baseURL: string;
   className?: string;
+  fromAdmin?: boolean;
+  currentStep?: number;
+  setIsLoading?: Dispatch<SetStateAction<boolean>>;
   useQueryParam?: boolean;
 }) => {
   const router = useRouter();
   return (
     <form
-      id="wizard-step-2"
-      name="wizard-step-2"
+      id={`wizard-step-${currentStep}`}
+      name={`wizard-step-${currentStep}`}
       onSubmit={(e) => {
         e.preventDefault();
+        setIsLoading && setIsLoading(true);
         router.replace("/");
       }}>
       <AppCategoryNavigation
         baseURL={baseURL}
-        fromAdmin
+        fromAdmin={fromAdmin}
         useQueryParam={useQueryParam}
-        containerClassname="w-full xl:mx-5 xl:w-2/3 xl:pr-5"
+        containerClassname={classNames(
+          "w-full xl:mx-5 xl:w-2/3 xl:pr-3.5",
+          fromAdmin ? " max-h-[400px] overflow-y-auto" : "max-w-[500px]"
```

Comment:
This is to have the app list within the admin wizard, contained inside the content section of the wizard and not expand its dimension, should be self-contained there.